### PR TITLE
new login endpoint without tokens

### PR DESCRIPTION
**Pull request by simionescuraluca**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,48 +25,46 @@
...
 			<version>3.6.3</version>
 		</dependency>
-		
+
 		<dependency>
 			<groupId>org.modelmapper</groupId>
 			<artifactId>modelmapper</artifactId>
 			<version>2.3.2</version>
 		</dependency>
-
```
Comment (dragosslujeru) [inline pom.xml]: revert unintended changes in pom

File: src/main/resources/changelogs/0001-address-migration.xml [hunk 2]
```diff
@@ -123,9 +123,11 @@
 		<preConditions onFail="MARK_RAN">
 			<sqlCheck expectedResult="NO">
 				SELECT is_Nullable
-				FROM INFORMATION_SCHEMA.COLUMNS
+				FROM
```
Comment (dragosslujeru) [inline src/main/resources/changelogs/0001-address-migration.xml]: revert unintended changes in liquibase

File: pom.xml (hunk 1 above)
Comment (simionescuraluca) [reply pom.xml]: done

File: src/main/resources/changelogs/0001-address-migration.xml (hunk 2 above)
Comment (simionescuraluca) [reply src/main/resources/changelogs/0001-address-migration.xml]: done

**Review by dragosslujeru — APPROVED** [review]

**Pull request merged**